packages/launchpad/cypress/e2e/onboarding-flow.cy.ts

```diff
@@ -2,36 +2,6 @@ describe('Launchpad: Onboarding Flow', () => {
   beforeEach(() => {
     cy.scaffoldProject('pristine')
     cy.openProject('pristine')
-    cy.withCtx((ctx, o) => {
-      ctx.actions.file.writeFileInProject('node_modules/cypress/package.json', JSON.stringify({
-        name: 'cypress',
-        main: 'index.js',
-      }))
-
-      ctx.actions.file.writeFileInProject('node_modules/@cypress/webpack-dev-server/package.json', JSON.stringify({
-        name: '@cypress/webpack-dev-server',
-        main: 'index.js',
-      }))
-
-      ctx.actions.file.writeFileInProject('node_modules/cypress/index.js', `
```

Comment:
No longer needed now that we're scaffolding the e2e tests w/ symlinked common `node_modules` in the same way we are system tests.